### ISSUE TITLE
Is Feedbunch dead?

### ISSUE THREAD

**Issue by ploguus** [amatriain/feedbunch#21]
Been having issues for several days, and now I can't export my feeds OPML.... Help?

**Comment by amatriain** [comment]
A server was having memory issues and it had slipped undetected. Should be fixed now. Can you try exporting your subscriptions again? Sorry for the inconvenience.

**Comment by ploguus** [comment]
I was able to export OPML, thank you. Something is still preventing the site from working for me, however. But now I can use a different reader for the feeds, at least.

**Comment by amatriain** [comment]
What error are you experiencing, exactly? What do you mean by "preventing the site from working"? I'd like to try to fix it, but I'm not aware of any current issues.

**Comment by ploguus** [comment]
Sometimes when I try to load the page, it just spins and spins and spins. Other times is crashes entirely on loading. This started about a week ago. Of course now that I want to grab a screenshot of the error, I can't replicate it. 🙄 Maybe it's fixed for me.

**Comment by ploguus** [comment]
![Annotation 2019-07-23 091114](https://user-images.githubusercontent.com/22798496/61723748-e04e4100-ad29-11e9-98c9-3173d5c661f1.jpg)

**Comment by amatriain** [comment]
I've found that the problems with background jobs, which blocked you from exporting your OPML, were deeper than I thought. Even after fixing the memory issues a Redis database had a bad corruption problem, probably caused by the memory issues in the first place. I've recreated the Redis instance from scratch and background jobs are again running smoothly.

I'm investigating what may cause those 522 errors you're seeing. For the moment I'm unable to reproduce them.

**Comment by amatriain** [comment]
I'm afraid I can't find the cause for those 522 errors. Feedbunch logs show no errors at the time the error in that screenshot happened, the service was up and running fine. I've checked Cloudflare logs as well, but they give too little information to be useful. The only thing I can think of is a temporary network error either in Cloudflare or the datacenter that hosts FeedBunch servers, but I cannot be sure.

In any case it looks like a transient error, I haven't been able to reproduce it. Closing the issue for now, might reopen if it happens again.
